lib/cdo/redshift_import.rb

```diff
@@ -16,8 +17,14 @@ def self.complete_table_import(schemas)
         target_table = import_table.partition(TEMP_TABLE_PREFIX).last
         backup_table = BACKUP_TABLE_PREFIX + target_table
 
-        # Rename existing table to back it up.
-        rename_table(schema, target_table, backup_table)
+        # Rename existing table to back it up, if it exists.
```

Comment:
No way to just check if a table exists, instead of rescuing the error?

Reply:
Redshift [doesn't seem to support](https://docs.aws.amazon.com/redshift/latest/dg/r_ALTER_TABLE.html) `ALTER TABLE ... IF EXISTS` which the latest versions of PostgreSQL [support](https://www.postgresql.org/docs/9.6/sql-altertable.html).  That seems leave two options:

1. Modify the query in `rename_table` to check for the table and then do conditional logic in the SQL to skip the ALTER TABLE
2. Create a dedicated Ruby method (`table_exists?`) that looks for the table and then use Ruby logic to determine whether to skip the call to `rename_table`.

Right now I favor rescuing the very specific UndefinedTable error.  I did move that logic inside the `rename_table` method since this logic applies to anytime we try to rename a table.